Source/WebCore/rendering/svg/SVGRenderSupport.cpp

```diff
@@ -167,14 +164,35 @@ void SVGRenderSupport::computeContainerBoundingBoxes(const RenderElement& contai
         const AffineTransform& transform = current.localToParentTransform();
         if (transform.isIdentity()) {
             updateObjectBoundingBox(objectBoundingBox, objectBoundingBoxValid, &current, current.objectBoundingBox());
-            strokeBoundingBox.unite(current.repaintRectInLocalCoordinates());
+            repaintBoundingBox.unite(current.repaintRectInLocalCoordinates());
         } else {
             updateObjectBoundingBox(objectBoundingBox, objectBoundingBoxValid, &current, transform.mapRect(current.objectBoundingBox()));
-            strokeBoundingBox.unite(transform.mapRect(current.repaintRectInLocalCoordinates()));
+            repaintBoundingBox.unite(transform.mapRect(current.repaintRectInLocalCoordinates()));
         }
     }
+}
+
+void SVGRenderSupport::computeContainerStrokeBoundingBox(const RenderElement& container, FloatRect& strokeBoundingBox)
+{
+    // When computing the strokeBoundingBox, we use the repaintRects (for hit-testing) of the container's children so that the container's stroke includes
```

Comment:
Not a new comment, but here it's clear that there's confusing between the stroke bounds, and additional inflation added by filters.

Reply:
Yeah, this is coming from the current code.